script.sh

```diff
@@ -147,7 +146,41 @@ case "${TRAVIS_DEBIAN_DISTRIBUTION}" in
 		;;
 esac
 
-# TRAVIS_DEBIAN_DISTRIBUTION is now set, so set dependent options
+# detect derivatives
+if [ "${TRAVIS_DEBIAN_DERIVATIVE:-}" = "" ]
+then
+	# strip derivate common pockets
+	for suffix in proposed updates security
+	do
+		TRAVIS_DEBIAN_DISTRIBUTION="${TRAVIS_DEBIAN_DISTRIBUTION%%-$suffix}"
+	done
+
+	case "${TRAVIS_DEBIAN_DISTRIBUTION}" in
+		precise)
```

Comment:
You can write this nicer as "pricise|trusty|xenial....)", no? :)

Reply:
correct, doing so :)

Thanks for the review!